src/analysis/normed/group/pointwise.lean

```diff
@@ -48,46 +44,70 @@ begin
   ... ≤ Rs + Rt : add_le_add (hRs x hx) (hRt y hy)
 end
 
-@[simp] lemma singleton_add_ball (x y : E) (r : ℝ) :
-  {x} + ball y r = ball (x + y) r :=
+@[simp] lemma singleton_add_ball (x y : E) (δ : ℝ) : {x} + ball y δ = ball (x + y) δ :=
 by simp only [preimage_add_ball, image_add_left, singleton_add, sub_neg_eq_add, add_comm y x]
 
-@[simp] lemma ball_add_singleton (x y : E) (r : ℝ) :
-  ball x r + {y} = ball (x + y) r :=
-by simp [add_comm _ {y}, add_comm y]
-
-lemma singleton_add_ball_zero (x : E) (r : ℝ) :
-  {x} + ball 0 r = ball x r :=
-by simp
+@[simp] lemma ball_add_singleton (x y : E) (δ : ℝ) : ball x δ + {y} = ball (x + y) δ :=
+by rw [add_comm, singleton_add_ball, add_comm y]
 
-lemma ball_zero_add_singleton (x : E) (r : ℝ) :
-  ball 0 r + {x} = ball x r :=
-by simp
+@[simp] lemma singleton_add_ball_zero (x : E) (δ : ℝ) : {x} + ball 0 δ = ball x δ := by simp
```

Comment:
You shouldn't need simp attributes on this one and the next if you are already proving them by simp.